site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleForm/TemplateScheduleForm.tsx

```diff
@@ -185,13 +204,31 @@ export const TemplateScheduleForm: FC<TemplateScheduleForm> = ({
         ? form.values.time_til_dormant_autodelete_ms * MS_DAY_CONVERSION
         : undefined,
 
+      autostop_requirement: {
+          days_of_week: calculateAutostopRequirementDaysValue(form.values.autostop_requirement_days_of_week),
+          weeks: autostop_requirement_weeks,
+      },
+
       allow_user_autostart: form.values.allow_user_autostart,
       allow_user_autostop: form.values.allow_user_autostop,
       update_workspace_last_used_at: form.values.update_workspace_last_used_at,
       update_workspace_dormant_at: form.values.update_workspace_dormant_at,
     })
   }
 
+  // Set autostop_requirement weeks to 1 when days_of_week is set to "off" or
+  // "daily".
```

Comment:
I would rather have a comment explaining why we need to set the weeks to 1 when `days_of_week` is set to `"off"` instead of what it is doing or keep both.